My Series List/My Series List/Model/Repository/RepositoryCoreData.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CoreData.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 19/11/2021.
+//
+
+import Foundation
+import CoreData
+import UIKit
+
+class RepositoryCoreData {
+   
+    var coreDataSeriesArray = [CoreDataSeries]()
+
+    func addSeries(id: Int32, name: String, overview: String, poster_path: String, vote_average: Double) {
```

Comment:
Consider adding a completionHandler here that allows to return an error message

Reply:
Should I use completionHandler as it isn't asynchronous task (I think) or a try catch would be enough ?